core/src/main/java/org/elasticsearch/common/settings/Setting.java

```diff
@@ -122,7 +122,7 @@ private Setting(Key key, @Nullable Setting<T> fallbackSetting, Function<Settings
         this.defaultValue = defaultValue;
         this.parser = parser;
         if (properties == null) {
-            throw new IllegalArgumentException("properties can not be null for setting [" + key + "]");
+            throw new IllegalArgumentException("properties cannot be null for setting [" + key + "]");
```

Comment:
Just curious, but what's the grammar error here? I thought that "cannot" was just a short form of "can not" and can be used interchangeably?